docs/Tutorial.md

```diff
@@ -21,6 +21,8 @@ Here is an overview of the result:
 React-admin uses React. We'll use [create-react-admin](https://github.com/marmelab/react-admin/tree/master/packages/create-react-admin) to bootstrap a new admin:
 
 ```sh
+npm create react-admin test-admin
```

Comment:
no, this won't work.

Reply:
I have tested it and it does actually

Reply:
It must be a recent addition to npm

Reply:
`npm create` [has been introduced in node 16](https://docs.npmjs.com/cli/v8/commands/npm-init?v=true) (npm 8), two years ago. By using it, we risk confusing people wondering width Node 14 or 12, wondering qhy it doesn't work. 

I suggest using `npm init` instead - this is supposed to work since way longer (https://docs.npmjs.com/cli/v6/commands/npm-init?v=true). `npm create` is actually an alias of `npm init`.

Reply:
I tested: both works. So no problem to use init instead of create but should we support, for the tutorial, node versions (12, 14 16 soon) which are end of life?

Reply:
> should we support, for the tutorial, node versions (12, 14 16 soon) which are end of life?

yes: nowhere in the doc we mention that we have minimal Node js requirements.